### DISCUSSION TITLE
Move recentlyadded to a own link

### DISCUSSION THREAD

**Discussion by 9Mad-Max5** [seanap/auto-m4b#9] [Ideas]
Hi,

I think it would make sense to move the recentlyadded folder to a different location.
On this way, it is possible to have the download container outside the usual docker environment.

Maybe it would make sense to chat on gitter or discord.

**Comment by seanap** [comment]
Hey,

I don't disagree, the recentlyadded folder should be able to be anywhere for ease of integration into the wide variety peoples setups.  My /temp folder is just in an smb share of my nas, which makes having 1 folder that contains everything pretty straight forward.  

Would it be worth breaking out the volume mapping for the Input (recentlyadded) and Output (untagged) folders? Probably would be less confusing for people to initially configure.

I am on either, feel free to continue the conversation anywhere.
